cmd/ctl/go.mod

```diff
@@ -12,45 +12,45 @@ go 1.20
 // or a branch name (master).
 
 require (
-	github.com/cert-manager/cert-manager v1.13.0-alpha.0.0.20230801130528-b93ec2f8242b
+	github.com/cert-manager/cert-manager v1.13.0-alpha.0.0.20230825134310-63cf4e0b1c56
```

Comment:
This will need to be updated again when you do another alpha or beta release, right?

> 9.  In this step, we make sure the Go module github.com/cert-manager/cert-manager/cmd/cmctl can be imported by third-parties.
> -- https://cert-manager.io/docs/contributing/release-process/#minor-releases

Reply:
Indeed. It does not hurt however to already upgrade this now too.